sdk/translation/azure-ai-translation-text/azure/ai/translation/text/_patch.py

```diff
@@ -105,8 +113,12 @@ def set_authentication_policy(credential, kwargs):
                         """Both 'resource_id' and 'region' must be provided with a TokenCredential for
                          regional resource authentication."""
                     )
+                scope: str = kwargs.pop("audience", DEFAULT_ENTRA_ID_SCOPE)
+                if not is_cognitive_services_scope(scope):
+                    scope = scope.rstrip("/") + DEFAULT_SCOPE
```

Comment:
should we concatenating the `./default` here? Isn't it possible the user could pass it as "https://cognitiveservices.azure.com/.default" and we would add an additional ".default" on it?

Reply:
OK, added.